test/runtime/connectivity.go

```diff
@@ -188,8 +188,23 @@ var _ = Describe("RuntimeValidatedConnectivityTest", func() {
 			dockerImage = "busybox:latest"
 			cniServer   = "cni-server"
 			cniClient   = "cni-client"
+			netDPath    = "/etc/cni/net.d/"
+			tmpDir      *helpers.CmdRes
 		)
-		var tmpDir *helpers.CmdRes
+
+		BeforeAll(func() {
+			// Remove any CNI plugin installed in the provision server. This
+			// helps to avoid issues on installing the new CNI
+			_ = vm.ExecWithSudo(fmt.Sprintf("rm -rf %[1]s/*.conf", netDPath)).ExpectSuccess(
```

Comment:
Ideally it shouldn't, but does this affect cri-o support within the VM after the test is ran? Furthermore, if users run this locally (which is a supported option as specified in the Ginkgo contributing guides), we may be removing configuration files that users have installed locally.